one_fm/utils.py

```diff
@@ -3843,7 +3843,8 @@ def set_out_of_office_for_leaves():
             if today == add_days(to_date, 1):
                 disable_out_of_office(employee_email)
 
-def update_active_employees_assurance_level():
```

Comment:
you changed the name, is this intentional, @samdanikouser

Reply:
@NoBoneZ , yes the function update_active_employees_assurance_level is called from the scheduler event of hooks.py file